### DISCUSSION TITLE
Hosting static files

### DISCUSSION THREAD

**Discussion by C-o-d-e-C-o-w-b-o-y** [vercel/community#564] [Help] [answered]
Hey all, hope you are well

I had a read thru the static image docs, and while the tech is great, I cant think of a way to host static resources with a persistent URL that doesn't change between deployments. The reason for this is it needs to be embedded on another site so effectively I need to use vercel as a (very mini, for like 100 images) S3 bucket. 

Any ideas? <3

**Comment by swarnava** [comment, marked as answer]
Hi there,

Are you referring to preview url? Because Production url (domain.vercel.app or yourcustomdomain.com) should always remain persistant. i.e if you host an image with path domain.vercel.app/path/to/your/image.png, it should remain same url forever unless you change the path otherwise.
